circleci/examples/inlineorb_config.yaml

```diff
@@ -0,0 +1,70 @@
+# An inline orb for common groups of Gradle commands
```

Comment:
I don't fully get what this comment means. Can you expound a bit.  Otherwise good to go.

Reply:
Added some more context to the comment 👍